plugins/src/core/index.ts

```diff
@@ -0,0 +1,369 @@
+import * as cordova from "cordova";
+import {
+  PushNotificationHandler,
+  InAppDeepLinkHandler,
+  InAppInboxUpdatedHandler,
+  DeepLinkHandler,
+} from "./handlers";
+
+export interface OptimoveConfig{
+  pushReceivedHandler?: PushNotificationHandler | null;
```

Comment:
are these not optional? They are always null or a function

Reply:
changed the handlers to non - optional